nansat_wkv.yml

```diff
@@ -491,3 +491,20 @@
   units: celsius
   minmax: -2 40
   colormap: jet
+
+- standard_name: y_sea_water_velocity
+  long_name: Y Sea Water Velocity
+  short_name: V
+  units: m s-1
+  units: m s-1
```

Comment:
Oops, now the units are duplicated. Both here and below. I accidentally included them in the suggestion.
```suggestion
```